c++/triqs_Nevanlinna/CMakeLists.txt

```
@@ -31,7 +31,7 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} DESTINATION include FILES_MATCHING
 
 # ========= Library Dependencies ==========
 
-find_package(Eigen3 REQUIRED)
+find_package (Eigen3 3.4 REQUIRED CONFIG)
 target_link_libraries(${PROJECT_NAME}_c PUBLIC Eigen3::Eigen)
 
 find_package(Boost REQUIRED)
```

Also needs at least Boost 1.73.0 because `std::unary_function` was removed in C++17.

```suggestion
find_package(Boost 1.73 REQUIRED)
```

So Ubuntu 20.04 is finally out of support for TRIQS, because its Boost is too old.

---

Thank you for pointing this out, updated the line accordingly